### PR TITLE
Replace PrivilegedCommand with ansible

### PR DESCRIPTION
**Pull request by millerdev**
Also fix a few bugs in the `copy-files` command.

:blowfish: Review by commit.

##### ENVIRONMENTS AFFECTED
None.

### CODE REVIEW

File: src/commcare_cloud/commands/migrations/copy_files.py [hunk 1]
```diff
@@ -218,12 +220,10 @@ def prepare_file_copy_scripts(target_host, source_file_configs, script_root):
 
 def copy_scripts_to_target_host(target_host, script_root, environment, ansible_context):
     local_files_path = os.path.join(script_root, target_host)
-
     destination_path = os.path.join('/tmp', REMOTE_MIGRATION_ROOT)
```
Comment (gherceg) [inline src/commcare_cloud/commands/migrations/copy_files.py:223]: I don't love that `destination_path` is defined both here and within `remove_scripts` as it seems prone to error. Maybe adding it as an optional parameter to `remove_scripts` could work?

**Review by gherceg — COMMENTED** [review]
Holding off on approval until one comment is resolved.

File: src/commcare_cloud/commands/migrations/copy_files.py (hunk 1 above)
Comment (millerdev) [reply src/commcare_cloud/commands/migrations/copy_files.py]: 27d6a00732dd3d9013db72eb2fa3721651959b16

Comment (gherceg) [reply src/commcare_cloud/commands/migrations/copy_files.py]: Even better!

**Pull request merged**